mabwiser/base_mab.py

```diff
@@ -22,6 +22,10 @@
 __version__ = __version__
 __copyright__ = __copyright__
 
+STATUS_TRAINED = 'trained'
```

Comment:
Replace this with `is_trained`?

Reply:
Yes, that's better. Same for `warm` to `is_warm`.